platform/src/main/java/org/hillview/storage/CassandraSSTableLoader.java

```diff
@@ -345,42 +295,42 @@ public ITable nonLazyLoading() {
                     i = 0;
                     /* Extracting the value for each column from a single row  */
                     for (ColumnData cd : row) {
-                        if (cd.column().isSimple()) {
-                            cell = (Cell) cd;
-                            cellType = cell.column().cellValueType();
-                            value = cellType.getSerializer().deserialize(cell.value());
-                        } else {
-                            throw new RuntimeException("[Hillview can't convert complex data]");
+                        /* Filter the column to only load the one marked true by columnToLoad[] */
```

Comment:
this comment is not very useful; instead you can document the function using JavaDoc and add a description of all arguments.